src/protocol.h

```diff
@@ -251,7 +251,7 @@ class CAddress : public CService
 {
 public:
     CAddress();
-    explicit CAddress(CService ipIn, uint64_t nServicesIn = NODE_NETWORK);
+    explicit CAddress(CService ipIn, uint64_t nServicesIn);
```

Comment:
@sipa  Have you made sure the test are not using the default?

I get a failure in `test/addrman_tests.cpp:71`


```cpp
test/addrman_tests.cpp: In member function ‘void addrman_tests::addrman_simple::test_method()’:
test/addrman_tests.cpp:71:31: error: no matching function for call to ‘CAddress::CAddress(CService&)’
     addrman.Add(CAddress(addr1), source);
                               ^